raiden-cli/src/raiden.ts

```diff
@@ -86,16 +86,13 @@ export default class RaidenService {
     let filteredChannels = this.allChannels;
 
     if (state) {
-      filteredChannels = _.filter(filteredChannels, (channel) => channel.state === state);
+      filteredChannels = filter(filteredChannels, (channel) => channel.state === state);
```

Comment:
Do you need lodash's `filter` for that? Why not simply `Array.prototype.filter`?

Reply:
:heavy_check_mark: